aerleon/lib/nokiasrl.py

```diff
@@ -62,9 +83,20 @@ class SRLTerm(openconfig.OCTerm):
 
     def SetAction(self) -> None:
         action = self.ACTION_MAP[self.term.action[0]]
-        self.term_dict['action'] = {action: {}}
+        log = {}
+        if self.term.logging:
+            if action == 'drop':
+                log = {"log": True}
+            else:
+                raise UnsupportedLogging(
+                    'logging can only be used with deny in term %s' % self.term.name
+                )
+        self.term_dict['action'] = {action: log}
 
-    def SetOptions(self) -> None:
+    def SetComments(self, comments: List[str]) -> None:
+        self.term_dict['description'] = "\n".join(comments)
+
+    def SetOptions(self, family: str) -> None:
```

Comment:
Family looks to be unused.

Reply:
It is used in 
```self.SetProtocol(family=family, protocol="udp")```

Needed because the attribute name is different for ipv4 versus ipv6